README.md

```diff
@@ -157,40 +167,38 @@ $ cp config/sample-settings.secret.yml config/settings.secret.yml
 Open the _config/settings.secret.yml_ file with a plain-text editor.
 
 1. Find the node named, "DATABASES" and set the
-database `NAME`, `USER`, and `PASSWORD` as appropriate. The result should resemble the following:
+database `NAME`, `USER`, and `PASSWORD` as appropriate. If you have a dump of an existing TolaActivity database, set the `NAME` to "tola_activty". The result should resemble the following:
 
     ```yaml
     DATABASES:
       default:
         ENGINE: "django.db.backends.mysql"
-        NAME: "<db_name>"
-        USER: "<db_username>"
-        PASSWORD: "SooperSekritWord"
+        NAME: "<db_name>" # "tola_activity"
+        USER: "<db_username>" # "tola"
+        PASSWORD: "<password>" # "SooperSekritWord"
         HOST: "localhost"
         PORT: ""
```

Comment:
@sanjuroj (sorry to keep pestering you by tagging you in all these, but I'm not 100% sure on most of what I'm proposing here) - my settings.secret has also `OPTIONS: {"init_command": "SET default_storage_engine=INNODB", 'charset': 'utf8mb4'}` and also test database options (`TEST: CHARSET: "utf8mb4" COLLATION: "utf8mb4_unicode_ci"`) but I can't remember if those were hail marys when we were trying to get utf8mb4 to work or if we still need those?

Reply:
I have those in my DB config as well.  They're also in the sample-settings.secret.yml, so if the instructions are to copy that file and update the NAME, USER, etc... settings, I think we're covered?

Reply:
I didn't change anything here.